### DISCUSSION TITLE
JavaScript not loading after livewire defer loading of my child component

### DISCUSSION THREAD

**Discussion by raowaqasmushtaq** [livewire/livewire#5394] [1. Help]
if i use defer loading or load component on any button click my JavaScripts working stop. how can i rescan my js and css in livewire?

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

**Reply by raowaqasmushtaq** [reply]
But i haven't found there any discussion about my issue. can you guid me about this issue why jquery not working while loading child component.

**Comment by joshhanley** [comment]
@raowaqasmushtaq I meant, you should have opened a discussion instead of an issue, hence why it got converted to a discussion (not saying that there is already a discussion about your problem 🙂).

Generally any third party javascript libraries that modify the DOM will have issues when running with Livewire. See here for more details https://laravel-livewire.com/docs/2.x/alpine-js#ignoring-dom-changes

Hope this helps!
